httpipfs.go

```diff
@@ -119,6 +131,30 @@ func (hi *HttpIpfs) ServeHTTP(res http.ResponseWriter, req *http.Request) {
 
 	selNode := unixfsnode.UnixFSPathSelectorBuilder(path.String(), dagScope.TerminalSelectorSpec(), false)
 
+	sig := RequestSignature{
+		RequestId: req.Header.Get("X-Request-Id"),
+		Cid:       rootCid.String(),
+		Protocol:  "https",
+	}
+	b, err := json.Marshal(sig)
+	if err != nil {
+		logError(http.StatusInternalServerError, err)
+		return
+	}
+	sigSigned, err := hi.privKey.Sign(b)
+	if err != nil {
+		logError(http.StatusInternalServerError, err)
+		return
+	}
+	res.Header().Set(
+		"X-Attestation",
+		fmt.Sprintf(
+			"\"%s.%s\"",
+			base64.StdEncoding.EncodeToString(b),
+			base64.StdEncoding.EncodeToString(sigSigned),
+		),
+	)
+
```

Comment:
We need to send the attestation in response trailer headers, after the retrieved content was transmitted.

I implemented in change in 5cf3fb9.

@juliangruber PTAL, could you also deploy my change to fly.io please?

Reply:
Strongly enough, GitHub did not pick my commit. The commit shows in our repo, but not in this PR 🤔

Reply:
ok, fixed via [334077a](https://github.com/ipld/frisbii/pull/10/commits/334077a2b3daca8bc1927325fc263f774e6a53e1)

Reply:
It turns now Fetch API does not allow clients to read HTTP trailers :(

Quoting from https://github.com/denoland/deno/issues/10214#issuecomment-821723392

> Trailers are not supported in the `fetch` spec anymore, and likely will not be re-introduced unless there is significant demand: [whatwg/fetch#981](https://github.com/whatwg/fetch/issues/981). I don't think we will be supporting them either if the web does not. Removal happened in [whatwg/fetch#772](https://github.com/whatwg/fetch/issues/772).

Let's not deploy my change yet.